### PR TITLE
Add support for parameters of type TimeSpan to SqlFilter

### LINKED ISSUE #6629: .net standard library does not allow using parameters of type TimeSpan
**Query/Question**
Wondering as to why the .net standard library does not support using parameters of type timeSpan for filters.

I wish to setup a filter that uses a DateTime message property and subtracts it from sys.EnqueuedDateTimeUtc, to determine if it should be processed as a high priority message or not. When 2 dates are subtracted, the result is a TimeSpan value in the filter, because of this the parameter that it needs to be compared to needs to be a TimeSpan. In .net framework it is possible to set a parameter of type TimeSpan, but with .net standard, this is failing.

***Why is this not a Bug or a feature Request?***
This could be a feature request, if the team is willing to make this change. Right now, making the assumption that the omission was intentional.

Please see: https://stackoverflow.com/questions/56654086/unable-to-set-timespan-based-parameter-on-an-azure-service-bus-subscription-filt for detailed information about the issue and why I am trying to use a TimeSpan parameter.

### PR DESCRIPTION
**Pull request by SeanFeldman**
Fixes #6629

### CODE REVIEW

File: sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/ManagementClientConstants.cs [hunk 1]
```diff
@@ -16,6 +16,7 @@ internal class ManagementClientConstants
         public const string SbNs = "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect";
         public const string XmlSchemaInstanceNs = "http://www.w3.org/2001/XMLSchema-instance";
         public const string XmlSchemaNs = "http://www.w3.org/2001/XMLSchema";
+        public const string SerializationNamespace = "http://schemas.microsoft.com/2003/10/Serialization/";
```
Comment (jsquire) [inline sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/ManagementClientConstants.cs]: nit:  The other members of this file use `Ns` where this uses `Namespace`; maybe we should consider unifying on one representation?   Personally, my preference goes to `Namespace`

**Review by jsquire — COMMENTED** [review]
Implementation looks good to me.   Since I lack a deeper context and understanding of how these changes may fit into the stack, I'm going to ask that @nemakam and @binzywu take ownership of approval here.

File: sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/ManagementClientConstants.cs (hunk 1 above)
Comment (SeanFeldman) [reply sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/ManagementClientConstants.cs]: I'm good with either @jsquire. Fully spelled out version is easier to read though. So if @nemakam agrees, I'll update the other two constant names despite being out of the context.

Comment (nemakam) [reply sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/ManagementClientConstants.cs]: I am good with either.
@SeanFeldman, you can take the call to go either way :)

**Comment by nemakam** [comment]
@SeanFeldman, The changes look good to me. Let me know whenever you are ready to merge, and I can merge it in!

**Review by nemakam — APPROVED** [review]

**Comment by axisc** [comment]
@jsquire , Can you merge the changes?

@nemakam  has approved.

**Comment by SeanFeldman** [comment]
@axisc I guess you and @jsquire didn't read through https://github.com/Azure/azure-sdk-for-net/pull/7325#discussion_r315283568 ... 🙂 

I'll issue a separate PR.
Lesson: when a PM says merge, don't rush 😹

**Comment by jsquire** [comment]
> @jsquire didn't read through #7325 (comment) ... 🙂

Truth.  I'm just making up for permissions still not quite being right in the repo by blindly pushing buttons on request.   Though, I forgot that Neeraj is empowered now and should have deferred.  Probably not the biggest mistake that I made yesterday.

File: sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/ManagementClientConstants.cs (hunk 1 above)
Comment (SeanFeldman) [reply sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/ManagementClientConstants.cs]: Will be addressed in https://github.com/Azure/azure-sdk-for-net/pull/7399

**Pull request merged**